src/applications/mhv/secure-messaging/components/Draft/DeleteDraft.jsx

```diff
@@ -45,7 +45,7 @@ const DeleteDraft = props => {
         text="Delete draft"
         secondary
         data-testid="delete-draft-button"
-        class="usa-button-secondary delete-draft-button vads-u-flex--1 vads-u-margin-bottom--1"
+        class="usa-button-secondary delete-draft-button vads-u-margin-bottom--1"

```

Comment:
I believe @KyleCardwell may be working on styling these. Just pointing out so that you guys can sync up or factor this in when resolving potential merge conflicts